lib/Backend/amd64/LinearScanMdA.S

```diff
@@ -0,0 +1,139 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+.intel_syntax noprefix
+#include "unixasmmacros.inc"
+
+
+#ifndef __APPLE__
+// BailOutRecord::BailOut(BailOutRecord const * bailOutRecord)
```

Comment:
Yes, originally I thought I needed them on Linux that's why they are under `ifndef __APPLE__`. I can remove `ifndef` and only keep the commented `.extern`s.